app/uk/gov/hmrc/helptosave/actors/UCThresholdManager.scala

```diff
@@ -185,9 +115,8 @@ object UCThresholdManager {
   case class GetThresholdValueResponse(result: Double)
 
   def props(thresholdConnectorProxy: ActorRef,
-            thresholdMongoProxy:     ActorRef,
             pagerDutyAlerting:       PagerDutyAlerting,
             scheduler:               Scheduler,
             config:                  Config): Props =
-    Props(new UCThresholdManager(thresholdConnectorProxy, thresholdMongoProxy, pagerDutyAlerting, scheduler, config))
+    Props(new UCThresholdManager(thresholdConnectorProxy, pagerDutyAlerting, scheduler, config))
```

Comment:
are you making use of `Command` in the above line `sealed trait Command` anywhere ?

Reply:
No so I have removed it, thank you